float32[] start
float32[] end
---
float32[] path_x
float32[] path_y
float32[] path_z 

